Test1/src/Problem1.kt

```diff
@@ -0,0 +1,11 @@
+fun main() {
+    val N = 10
+    var sum = 0
+    val arr = IntArray(N) { i -> i }
+    for(i in arr) {
+        if (i % 2 == 0) {
+            sum += arr[i]
+        }
+    }
```

Comment:
There a couple of [usefull functions for arrays in kotlin](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-array/) scout the docs maybe you can find a way to make this more idiomatic.